pulp_ansible/app/tasks/upload.py

```diff
@@ -75,12 +75,8 @@ def process_collection_artifact(artifact, namespace, name, version):
     return collection_info
 
 
-def finish_collection_upload(collection_version, tags, origin_repository):
-    """After CollectionVersion has been created update its tags and latest_version."""
-    for name in tags:
-        tag, created = Tag.objects.get_or_create(name=name)
-        collection_version.tags.add(tag)
-
+def finish_collection_upload(collection_version, origin_repository):
+    """After CollectionVersion has been created update its repository."""
```

Comment:
Think of any ideas for removing this workaround now that tags are solved?

Reply:
looking...

Reply:
I think the name clash is still not resolved and we need the workaround.
But some comments around it are more than outdated, and it is only that one thing left. I will fold it into the serializer create call. That should surround it with the better context to understand it.